float64 speed_left_target
float64 speed_right_target
uint16 gear_target

float64 speed_left
float64 speed_right
        
uint16 state_of_activation
float64 random_number
uint16 shift_value
uint16 activation_code
uint16 gear
float64 state_of_charge

bool running_read
bool running_write

uint16 state